gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Mar 14 15:49:42 EDT 2017
+#Thu Nov 14 17:42:41 WET 2024
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
```

Comment:
Shouldn't we point to a newer version than `7.0.2`? Also, shouldn't we point to an `all` distribution instead of a `bin`?